test/template-tests/testcases/deploymentTemplate/Virtual-Machines-Should-Not-Be-Preview.test.ps1

```diff
@@ -0,0 +1,21 @@
+﻿param(

```

Comment:
Can you run this test on the repo and see what kind of failures you pick up... to see if templates have some authoring techniques that throw unexpected errors.